<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dlg_select</class>
 <widget class="QDialog" name="dlg_select">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>385</width>
    <height>268</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>과목 선택 창</string>
  </property>
  <widget class="QListWidget" name="list_subject">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>31</y>
     <width>251</width>
     <height>181</height>
    </rect>
   </property>
   <property name="dragEnabled">
    <bool>true</bool>
   </property>
   <property name="dragDropMode">
    <enum>QAbstractItemView::InternalMove</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_add">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>230</y>
     <width>51</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>추가</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txt_addItem">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>240</y>
     <width>191</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>10</y>
     <width>64</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>Week </string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>220</y>
     <width>64</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>과목 입력 :</string>
   </property>
  </widget>
  <widget class="QListWidget" name="list_week">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>30</y>
     <width>61</width>
     <height>231</height>
    </rect>
   </property>
   <property name="resizeMode">
    <enum>QListView::Fixed</enum>
   </property>
   <property name="viewMode">
    <enum>QListView::ListMode</enum>
   </property>
   <property name="sortingEnabled">
    <bool>false</bool>
   </property>
   <item>
    <property name="text">
     <string>1</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>3</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>4</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>5</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>6</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>7</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>8</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>9</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>10</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>11</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>12</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>13</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>14</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>15</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>64</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>과목</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
